src/app/shell/personal-cabinet/parent/children/children.component.html

```diff
@@ -30,7 +30,7 @@
   <app-paginator
     [totalEntities]="childrenCards.totalAmount"
     [currentPage]="currentPage"
-    [itemsPerPage]="childrensPerPage$ | async"
+    [itemsPerPage]="itemsPerPage$ | async"
```

Comment:
revert this change please, because for children.component we should use the old pagination, there is not table here